src/configs/localeConfig.js

```diff
@@ -17,28 +17,32 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-
-import * as RNLocalize from 'react-native-localize';
-import { SUPPORTED_LANGUAGES, DEFAULT_LANGUAGE } from './config';
-
-
-const userPreferredLocales = RNLocalize.getLocales();
-const userPreferredLanguages = userPreferredLocales.map(({ languageCode }) => languageCode);
-
-const getSupportedLanguage = () => {
-  const language = userPreferredLanguages.find((languageCode) => SUPPORTED_LANGUAGES.includes(languageCode));
-  return language || DEFAULT_LANGUAGE;
-};
-
-const languageDetector = {
-  type: 'languageDetector',
-  async: true,
-  detect: (callback: (lang: string) => void) => {
-    callback(getSupportedLanguage());
+/* eslint-disable i18next/no-literal-string */
+
+import {
+  CAPITALIZATION_POSTPROCESSOR,
+  PUNCTUATION_POSTPROCESSOR,
+  SUFFIX_PREFIX_POSTPROCESSOR,
+} from 'constants/localesConstants';
+
+const EN_COMMON = require('../locales/en/common.json');
+const EN_AUTH = require('../locales/en/auth.json');
+
+const DEFAULT_LANGUAGE = 'en';
+const DEFAULT_NAMESPACE = 'common';
+
+export default {
+  isEnabled: true,
+  baseUrl: 'http://pillar-stories.dev.imas.lt/locales/', // todo: change into real one;
```

Comment:
No real one yet?

Reply:
Nope - not yet.
When approved, I'll remove it so only locally stored translations will be used.
Kept it here for testing / reviews only :)

I'll update this part in separate PR as we've later on decided on versioning.
Expected solution is explained here: https://linear.app/pillarproject/issue/PIL-842/implementation-language-injection